docs/site/paredit.md

```diff
@@ -46,6 +46,34 @@ The Paredit commands are sorted into **Navigation**, **Selection**, and **Edit**
 
 To make the command descriptions a bit clearer, each entry is animated. When you try to figure out what is going on in the GIFs, focus on where the cursor is at the start of the animation loop.
 
+### Command Args
+
+VSCode allows you to provide arguments to configured bindings, allowing, for example, different variants or behaviors for the same command, each bound to different shortcuts. Calva takes advantage of this by offering one official command with args:
```

Comment:
There are other Calva commands that accept arguments. Also, I tend to avoid making the Calva docs teach VS Code stuff, because ... well, that's for VS Code docs to do. (I admit there are places where we should do this anyway, but anyway. 😄 ) 

```suggestion
Some Paredit commands accept arguments. You can utilize this in keybindings and from [Joyride](https://github.com/BetterThanTomorrow/joyride).
```